samples/DirectX/D3D12/HelloConstBuffer12.cs

```diff
@@ -70,9 +70,9 @@ public unsafe class HelloConstBuffer12 : DX12Sample
         // App resources.
         private ID3D12Resource* _vertexBuffer;
         private D3D12_VERTEX_BUFFER_VIEW _vertexBufferView;
-        private ID3D12Resource* _constantBuffer;
+        private readonly ID3D12Resource* _constantBuffer;
```

Comment:
This being `readonly` means it isn't being disposed correctly.

Reply:
fixed.